Xamarin.PropertyEditing.Mac/Controls/EditorContainer.cs

```diff
@@ -24,6 +28,127 @@ public EditorViewModel ViewModel
+							midLineShape.Path = midLinePath.ToCGPath ();
+							midLineShape.FillColor = null;
+							midLineShape.Opacity = 1.0f;
+							midLineShape.StrokeColor = HostResources.GetNamedColor (NamedResources.ForegroundColor).CGColor;
+							Layer.AddSublayer (midLineShape);
+
+							this.variationLayersList.Add (midLineShape);
+						}
+
+
+						var deleteVariantButton = new CommandButton {
+							Bordered = false,
+							Command = propertyViewModel.RemoveVariationCommand,
+							// TODO Awaiting Image from Vaclav Image = HostResources.GetNamedImage ("pe-list-remove-16"),
+							Title = "x",
+							ToolTip = Properties.Resources.RemoveVariant,
+							TranslatesAutoresizingMaskIntoConstraints = false,
+						};
+
+						AddSubview (deleteVariantButton);
+						AddConstraints (new[] {
+							NSLayoutConstraint.Create (deleteVariantButton, NSLayoutAttribute.Top, NSLayoutRelation.Equal, this, NSLayoutAttribute.Top, 1f, 3f),
+							NSLayoutConstraint.Create (deleteVariantButton, NSLayoutAttribute.Left, NSLayoutRelation.Equal, this, NSLayoutAttribute.Left, 1f, 12f),
+							NSLayoutConstraint.Create (deleteVariantButton, NSLayoutAttribute.Width, NSLayoutRelation.Equal, 1f, 16f),
+						});
+
+						this.variationControlsList.Add (deleteVariantButton);
+
+						NSView previousControl = deleteVariantButton;
+						foreach (PropertyVariationOption item in propertyViewModel.Variation) {
+							var selectedVariationTextField = new UnfocusableTextField {
+								BackgroundColor = HostResources.GetNamedColor (NamedResources.FrameBoxButtonBackgroundColor),
+								Bordered = false,
+								Font = NSFont.SystemFontOfSize (NSFont.SystemFontSizeForControlSize (NSControlSize.Small) - 1f),
+								TranslatesAutoresizingMaskIntoConstraints = false,
+								StringValue = string.Format (" {0}: {1} ", item.Category, item.Name),
+							};
+
+							AddSubview (selectedVariationTextField);
+							AddConstraints (new[] {
+								NSLayoutConstraint.Create (selectedVariationTextField, NSLayoutAttribute.Top, NSLayoutRelation.Equal, this, NSLayoutAttribute.Top, 1f, 5f),
+								NSLayoutConstraint.Create (selectedVariationTextField, NSLayoutAttribute.Left, NSLayoutRelation.Equal, previousControl,  NSLayoutAttribute.Right, 1f, 3f),
+								NSLayoutConstraint.Create (selectedVariationTextField, NSLayoutAttribute.Height, NSLayoutRelation.Equal, 1f, 16f),
+							});
+
+							previousControl = selectedVariationTextField;
+							this.variationControlsList.Add (selectedVariationTextField);
+						}
+					} else {
+
+						var addVariantButton = new CommandButton {
+							Bordered = false,
+							Command = propertyViewModel.RequestCreateVariantCommand,
+							// TODO Awaiting Image from Vaclav Image = HostResources.GetNamedImage ("pe-list-add-16"),
+							Title = "+",
+							ToolTip = Properties.Resources.AddVariant,
+							TranslatesAutoresizingMaskIntoConstraints = false,
+						};
+
+						AddSubview (addVariantButton);
```

Comment:
```suggestion
						LeftEdgeView = addVariantButton;
```

This is what `LeftEdgeView` is meant for. When we add error handling, these two will draw over each other otherwise.

Reply:
OK